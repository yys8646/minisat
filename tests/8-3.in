8
.0......
...1....
..0...0.
.....00.
...0....
1..0....
.0......
1...1.00
