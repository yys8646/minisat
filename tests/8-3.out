8
00110011
10110010
01001101
10011001
01100110
11001010
00110101
11001100
